src/mumble/PulseAudio.cpp

```diff
@@ -301,6 +301,12 @@ void PulseAudioSystem::eventCallback(pa_mainloop_api *api, pa_defer_event *) {
 			qsInputCache = idev;
 
 			pa_stream_connect_record(pasInput, qPrintable(idev), &buff, PA_STREAM_ADJUST_LATENCY);
+
+			// Ensure stream is initially un-muted
+			pa_stream_cork(pasInput, 0, NULL, NULL);
+
+			// connect PulseAudioInput corkAudioInputStream signal to here
+			QObject::connect(g.mw, SIGNAL(corkAudioInputStream(const bool)), this, SLOT(corkAudioInputStream(const bool)), Qt::QueuedConnection);
```

Comment:
```suggestion
			connect(g.mw, MainWindow::corkAudioInputStream, this, PulseAudioSystem::corkAudioInputStream, Qt::QueuedConnection);
```

Reply:
Why did you choose `Qt::QueuedConnection`?

Reply:
I presume you mean

```suggestion
connect(g.mw, &MainWindow::corkAudioInputStream, this, &PulseAudioSystem::corkAudioInputStream, Qt::QueuedConnection)
```

instead?

To be honest, I'm not sure why I used `Qt::QueuedConnection`. I first authored this PR in 2013, and any of my motivations that were not written down have been lost to time.

I might have been worried about this signal being dispatched from code which held [`pa_threaded_mainloop_lock()`](https://freedesktop.org/software/pulseaudio/doxygen/thread-mainloop_8h.html#a2fd002f510ed24cabacb63a26bf4c977) already… but the docs say that this lock is re-entrant / recursive, so that is not an issue.

I don't immediately see any reason why we couldn't use `Qt::AutoConnection` here, but I must also admit to not being terribly familiar with Qt. I don't use it during my day-to-day activities. Would you like me to switch to `Qt::AutoConnection`?

Reply:
> I presume you mean
> `connect(g.mw, &MainWindow::corkAudioInputStream, this, &PulseAudioSystem::corkAudioInputStream, Qt::QueuedConnection)`
>
> instead?

Yes, sorry.

> To be honest, I'm not sure why I used `Qt::QueuedConnection`. I first authored this PR in 2013, and any of my motivations that were not written down have been lost to time.
>
> I might have been worried about this signal being dispatched from code which held [`pa_threaded_mainloop_lock()`](https://freedesktop.org/software/pulseaudio/doxygen/thread-mainloop_8h.html#a2fd002f510ed24cabacb63a26bf4c977) already… but the docs say that this lock is re-entrant / recursive, so that is not an issue.
>
> I don't immediately see any reason why we couldn't use `Qt::AutoConnection` here, but I must also admit to not being terribly familiar with Qt. I don't use it during my day-to-day activities. Would you like me to switch to `Qt::AutoConnection`?

I would simply omit the argument (`Qt::AutoConnection` is the default).

https://doc.qt.io/qt-5/qt.html#ConnectionType-enum

Reply:
I have tested 7bfcd623f07dfb27170f4ff9ae9f4f62afa3fc70 with `AutoConnection`, and this patch appears to work just fine with that change. Thanks for the suggestion!

Reply:
You're welcome!